pkg/controller/networkpolicy/clustergroup.go

```diff
@@ -183,10 +208,10 @@ func (n *NetworkPolicyController) syncInternalGroup(key string) error {
 		klog.Infof("Didn't find the ClusterGroup %s, skip processing of internal group", grp.Name)
 		return nil
 	}
-	if grp.IPBlock == nil {
+	n.processServiceReference(grp)
```

Comment:
when the referenced Service is deleted, the processServiceReference sets the groups Selector to nil. L212 will evaluate to false and the internal group is not updated in store. how does it remove the previous group members from internal group?

Reply:
Thanks for spotting this... Fixed and added a UT for this case in the latest commit.